### ISSUE TITLE
npm install error

### ISSUE THREAD

**Issue by ForCrypto1101** [HashLips/hashlips_art_engine#728]
i was trying to make my collection and i got stucked when i tried to run npm install. I have no coding knowledge so i would want to know what should i do.

npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 
npm ERR! code 1
npm ERR! path /Users/111imac/Documents/hashlips_art_engine-main/node_modules/canvas
npm ERR! command failed

**Comment by danielHype** [comment]
Try npm install -g node-pre-gyp

**Comment by meanthadar-p** [comment]
Same here, but I found old issue that already solved ref: https://github.com/HashLips/hashlips_art_engine/issues/621

it's work for me. hope it work to you too :))
@ForCrypto1101 @cRhettiarachchi

**Comment by ForCrypto1101** [comment]
I think its probably because of the 1 chip

**Comment by cRhettiarachchi** [comment]
Yeah, the problem arises from an npm dependency not supported on the M1 chip.

**Comment by bolshoytoster** [comment]
@cRhettiarachchi [you might be able to find some workarounds though.](https://github.com/HashLips/hashlips_art_engine/issues?q=is%3Aissue+M1)
